spk/zerotier/Makefile

```diff
@@ -0,0 +1,35 @@
+SPK_NAME = zerotier
+SPK_VERS = 1.2.12
+SPK_REV = 0
+SPK_ICON = src/zerotier.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = Joseph Henry @ ZeroTier, Inc.
+DESCRIPTION = A Simple global P2P encrypted LAN for all of your devices.
+CHANGELOG = Various performance upgrades and bugfixes to core service.
+HOMEPAGE = https://my.zerotier.com
+REPORT_URL = https://github.com/zerotier/ZeroTierNAS/issues
+DISPLAY_NAME = ZeroTier
+
+DSM_UI_DIR = ui
+RELOAD_UI = yes
+OS_MIN_VERS = 6.0-7321
+FIRMWARE = 6.0-7321
+STARTABLE = yes
+
+LICENSE  = GPLv3
+INSTALLER_SCRIPT = src/installer.sh
```

Comment:
To reduce script volume, you should take benefits from generic installer and startup script: https://github.com/SynoCommunity/spksrc/wiki/Service-Support
For security reason, service should not run as root if not required.